src/tokens/solana.tokenlist.json

```diff
@@ -4968,6 +4968,18 @@
       "decimals": 0,
       "logoURI": "https://cdn.jsdelivr.net/gh/trustwallet/assets/blockchains/ethereum/assets/0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48/logo.png",
       "tags": []
+    },
+    {
+      "chainId": 101,
+      "address": "z3dn17yLaGMKffVogeFHQ9zWVcXgqgf3PQnDsNs2g6M",
+      "symbol": "OXY",
```

Comment:
This is already added here https://github.com/solana-labs/token-list/blob/main/src/tokens/solana.tokenlist.json#L425